docs/index.asciidoc

```diff
@@ -0,0 +1,102 @@
+:plugin: bytes
+:type: filter
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: %VERSION%
+:release_date: %RELEASE_DATE%
+:changelog_url: %CHANGELOG_URL%
+:include_path: ../../../../logstash/docs/include
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+[id="plugins-{type}s-{plugin}"]
+
+=== Bytes filter plugin
+
+include::{include_path}/plugin_header.asciidoc[]
+
+==== Description
+
+Parse string representations of computer storage sizes such as "123 MB" or "5.6Gb" into
+their numeric value in bytes.
+
+This plugin understands: 
+
+- bytes ("B")
+- kilobytes ("KB" or "kB")
+- megabytes ("MB", "mb", or "mB")
+- gigabytes ("GB", "gb", or "gB")
+- terabytes ("TB", "tb", or "tB")
+- petabytes ("PB", "pb", or "pB")
+- exabytes ("EB", "eb", or "eB")
+
+It also recognizes units as bits.
+
+- bits ("b")
+- kilobits ("kb" or "Kb")
+- megabits ("Mb")
+- gigabits ("Gb")
+- terabits ("Tb")
+- petabits ("Gb")
+- exabits ("Eb")
+
+
+[id="plugins-{type}s-{plugin}-examples"]
+==== Examples
+
+
+[cols="<,<",options="header",]
+|=======================================================================
+|Input string |Numeric value in bytes
+|40|40
+|40B|40
+|40 B|40
+|40b|5
+|40KB|40960
+|40kB|40960
+|40.5KB|41472
```

Comment:
nit: this 40.5 example is bit out of place...the others are very nice.

Reply:
Ah, I wanted to have an example that shows decimal usage.